src/main/java/frc/robot/Robot.java

```diff
@@ -57,6 +63,9 @@ public void disabledPeriodic() {}
   @Override
   public void autonomousInit() {
     m_autonomousCommand = m_robotContainer.getAutonomousCommand();
+    m_robotContainer.getOdometry().zeroHeading();
```

Comment:
The reason `zeroHeading` doesn't work is because you're not actually calling it before teleop, you're only calling it before autonomous.

Reply:
No I just forgot to remove that one, we had another function in teleop init that didn't work as well. I also had one in the robot container and odometry and neither of those worked. I think the issue was something to do with the recalibration time or something